DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -48,17 +57,32 @@ struct NetworkProtectionStatusView: View {
                     }
                 ))
                 .disabled(statusModel.shouldShowLoading)
-                .toggleStyle(SwitchToggleStyle(tint: Color(designSystemColor: .accent)))
+                .toggleStyle(SwitchToggleStyle(tint: .toggleColor))
             }
+            .background(Color.cellBackground)
+        } header: {
             HStack {
-                if let status = statusModel.statusMessage {
-                    Text(status)
-                        .foregroundColor(statusModel.isNetPEnabled ? .green : .red)
+                Spacer()
+                VStack(alignment: .center, spacing: 16) {
+                    Image(statusModel.statusImageID)
+                        .resizable()
+                        .scaledToFit()
+                        .frame(height: 96)
+                        .padding(8)
+                    Text(statusModel.headerTitle)
+                        .font(.system(size: 17, weight: .semibold))
+                        .multilineTextAlignment(.center)
+                        .foregroundColor(.titleText)
+                    Text(UserText.netPStatusHeaderMessage)
```

Comment:
I'm sharing this with you because I didn't know this until yesterday, and because I have the doubt that the font is correct.  I'm not sure if `.system(...)` returns "SF Pro", so apologies if it does.

You can get the SwiftUI styling code directly from Figma, as shown below.

<img width="964" alt="Screenshot 2023-08-10 at 14 06 52" src="https://github.com/duckduckgo/iOS/assets/1836005/21fb15fe-9e7b-4f13-8c5a-28b0080b4283">

Reply:
Yeah, system does return that. But this is still useful as I did not know about that. Thanks!